### PR TITLE
[Refactor](RF) refactor the profile of rf and pipeline-x support local ignore

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

File: be/src/exprs/runtime_filter.cpp [hunk 1]
```diff
@@ -1032,8 +1019,8 @@ Status IRuntimeFilter::push_to_remote(const TNetworkAddress* addr, bool opt_remo
     pquery_id->set_lo(_state->query_id.lo());
 
     auto pfragment_instance_id = merge_filter_request->mutable_fragment_instance_id();
-    pfragment_instance_id->set_hi(_state->fragment_instance_id().hi());
-    pfragment_instance_id->set_lo(_state->fragment_instance_id().lo());
+    pfragment_instance_id->set_hi(BackendOptions::get_local_backend().id);
```
Comment (yiguolei) [inline be/src/exprs/runtime_filter.cpp:1022]: 如果再系统启动的时候，backend id 没有设置，此时是0， 会不会有问题？

Comment (HappenLee) [reply be/src/exprs/runtime_filter.cpp:1022]: 没有第一次成功的心跳，fe不会调度的

**Review by yiguolei — APPROVED** [review]

**Pull request merged**